lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -287,10 +291,17 @@ def assume_top_position
           set_list_position(acts_as_list_top)
         end
 
+        # Forces item to temporary position that will not conflict with other items
+        def assume_temporary_position
+          set_list_position(bottom_position_in_list(self).to_i + 2)
+        end
+
         # This has the effect of moving all the higher items down one.
-        def increment_positions_on_higher_items
+        def increment_positions_on_higher_items(position)
           return unless in_list?
-          acts_as_list_list.where("#{quoted_position_column_with_table_name} < ?", send(position_column).to_i).increment_all
+          safe_increment_all(
+            acts_as_list_list.where("#{quoted_position_column_with_table_name} < ?", position)
```

Comment:
@dkubb the expression `where.(#{quoted_position_column_with_table_name} < ?", position)` gets repeated at least 3 times in this diff. Should we move this to a method / scope?